packages/gatsby/src/schema/build-connection-fields.js

```diff
@@ -104,7 +104,30 @@ module.exports = type => {
           groupConnections.push(groupConn)
         })
 
-        return groupConnections
+        // make tags arrays case ignored
+        const tags = groupConnections.map(e=>{
```

Comment:
this one tho might illustrate the potential a bit better tthough: https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-transformer-react-docgen/src/extend-node-type.js